tree/tree/src/TChain.cxx

```diff
@@ -1401,7 +1401,7 @@ Long64_t TChain::LoadTree(Long64_t entry)
             }
             // Notify user if requested.
             if (fNotify) {
-               fNotify->Notify();
+               if(!fNotify->Notify()) return -5;
```

Comment:
Currently we have:
```
///   * -5: Internal error, please report the circumstance when this happen
///       as a ROOT issue.
```
Is that what you mean here?

[There is a missing white space after the if]

Reply:
It looked to me the cheapest choice.
Adding another code such as -6 seemed to be not too nice because -5 is the fallback error code. What would you go for?
(white space, +1)

Reply:
It is not an internal error but rather a user error in this case.  So would use -6.

However, I am not sure that LoadTree should 'fail' if one of the notify fails ....

Reply:
Why would it succeed? The problem to solve here (TTreeReader::SetEntryBase) is that in absence of a mechanism for propagating an error within the callback to the caller of LoadTree now one needs a state in the callback. And this case is an "easy" one. In principle, the caller of LoadTree might not have access to the callback and its state.